lib/engine/devices/database.js

```diff
@@ -25,6 +25,11 @@ import * as Tp from 'thingpedia';
 const ObjectSet = Tp.ObjectSet;
 import SyncDatabase from '../db/syncdb';
 
+// check for updates of all devices every 3 hours
+// this is to catch bug fixes quickly
+// we'll lower this to 24 hours after the alpha release
+const UPDATE_FREQUENCY = 60 * 1000; //3 * 3600 * 1000;
```

Comment:
isn't this 1 minute?

Reply:
Ugh yes I set this to a minute for testing, I forgot to revert back to 3 hours